Description: Removed unnecessary linkage with libXmu (-lXmu) - executable wasn't using any of the symbols.
Author: Alexandre Dantas <eu@alexdantas.net>
Last-Update: 2014-08-18
--- xbomb.orig/Makefile
+++ xbomb/Makefile
@@ -19,7 +19,7 @@ INCLUDES=
 
 LIB=
 
-XLIB=-L/usr/X11R6/lib -lXaw -lXmu -lXt -lX11
+XLIB=-L/usr/X11R6/lib -lXaw -lXt -lX11
 
 COMPILE=$(CC) -c $(CFLAGS)
 
